lib/shared/bucketing-assembly-script/assembly/helpers/jsonHelpers.ts

```diff
@@ -210,3 +212,20 @@ export function isFlatJSONObj(json: JSON.Obj | null): bool {
     }
     return true
 }
+
+export function getJSONObjFromCustomDataMap(customData: Map<string, CustomDataValue>): JSON.Obj {
```

Comment:
pretty straightforward converter to go from the Map of CustomDataValue that we get from decoding Protobuf, to a JSON Obj